jbpm-test-util/src/main/java/org/jbpm/test/persistence/util/ProcessCreatorForHelp.java

```diff
@@ -14,21 +14,25 @@
  * limitations under the License.
  */
 
-package org.jbpm.persistence.map.impl;
+package org.jbpm.test.persistence.util;
 
 import java.util.ArrayList;
 import java.util.List;
 
 import org.jbpm.process.core.Work;
-import org.jbpm.process.core.datatype.impl.type.ObjectDataType;
-import org.jbpm.process.core.impl.WorkImpl;
-import org.jbpm.persistence.session.objects.Person;
 import org.jbpm.process.core.context.variable.Variable;
+import org.jbpm.process.core.datatype.impl.type.ObjectDataType;
 import org.jbpm.process.core.event.EventTypeFilter;
+import org.jbpm.process.core.impl.WorkImpl;
 import org.jbpm.ruleflow.core.RuleFlowProcess;
+import org.jbpm.test.persistence.session.objects.Person;
 import org.jbpm.workflow.core.Node;
 import org.jbpm.workflow.core.impl.ConnectionImpl;
-import org.jbpm.workflow.core.node.*;
+import org.jbpm.workflow.core.node.EndNode;
+import org.jbpm.workflow.core.node.EventNode;
+import org.jbpm.workflow.core.node.StartNode;
+import org.jbpm.workflow.core.node.SubProcessNode;
+import org.jbpm.workflow.core.node.WorkItemNode;
 
 public class ProcessCreatorForHelp {
```

Comment:
Probably again used just in jbpm-persistence tests?

Reply:
This class is used by TestPersistenceContextBase class (one of the class we want to reuse) and it was located in the jbpm-installer module originally (now in the jbpm-test-util).

So this ProcessCreatorForHelp class is used by two classes, one in the jbpm-persistence  and one in the jbpm-test-util module.

I will keep it in jbpm-test-util. What do you think?

Reply:
Ah, then it's fine.